.github/workflows/release-container.yml

```diff
@@ -0,0 +1,45 @@
+  name: Release Keycloak container
+  on:
+    workflow_dispatch:
+    push:
+      tags:
+        - '[0-9]+.[0-9]+.[0-9]+'
+  jobs:
+    publish:
+      if: github.repository == 'keycloak/keycloak'
+      runs-on: ubuntu-latest
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        
+      - name: Container metadata
```

Comment:
Didn't knew about this `metadata-action` but looks great!

Talking about the implementation, from what I understand from the [documentation](https://github.com/docker/metadata-action#typesemver), this assumes a tag to be in the form: `v1.2.3` and not `1.2.3`, have you tested it without the `v` prefix already?
If this is the case you might want to tune the filtering regexp [here](https://github.com/keycloak/keycloak/pull/10683/files#diff-69a4610cce35bf32d579cc8d14aa59de81d80d23f0269d80861bceb81f57b176R6).

Reply:
Yup, tested it of course and it works just fine with "x.y.z" tags:
https://quay.io/repository/stianst/keycloak?tab=tags